lib/scrape.js

```diff
@@ -224,7 +233,12 @@ async function findGitHubUser(displayName, org) {
   const shortName = validUsername(displayName)
 
   const username = await findGitHubUserInOrg(displayName, org)
-  if (username) return username
+  if (username) {
+    return {
+      login: username,
+      updated: Date.now(),
```

Comment:
you're abusing this function.  A find function will always have an `updated` of "now", so it is redundant and annoying to be returning that to the caller.

the storing of cache information doesnt belong in this function.